src/components/repository/IntentsAndLabelsList.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <div>
+    <div class="button-position">
+      <bh-button
+        ref="intents"
+        :primary="active"
+        class="button-position__space"
+        size="medium"
+        @click="activeChange(true)">
+        Intents
+      </bh-button>
+      <bh-button
+        ref="labels"
+        :primary="!active"
+        class="button-position__space"
+        size="medium"
+        @click="activeChange(false)">
+        Labels
+      </bh-button>
+    </div>
+    <div v-if="active">
```

Comment:
Should be add class named as `intents-and-labels-list__list`

Reply:
Replace `active` for `intentsActive` or create data value `activeList` with possible values `intents` or `labels`.

E.g.:
```
<div v-if="activeList === 'intents'"></div>
<div v-if="activeList === 'labels'"></div>
```